browser/ui/views/tabs/brave_tab_style_views.inc.cc

```diff
@@ -294,6 +299,10 @@ TabStyle::SeparatorBounds BraveVerticalTabStyle::GetSeparatorBounds(
   origin.Scale(scale);
   bounds.trailing.Offset(-origin.x(), -origin.y());
 
+  if (is_next_tab_tiled) {
+    bounds.trailing = {};
```

Comment:
I think logic for controlling whether a separator is visible (given its state and its siblings' state) is mostly handled in `GetSeparatorOpacity`. Should we also move this "next tab tiled" logic there?

Reply:
Yes, that makes sense. Done.